### PR TITLE
Puzzle page feedback

### PR DESCRIPTION
**Pull request by Opeyem1a**

### CODE REVIEW

File: src/components/App/FeedbackGif/FeedbackGif.module.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+.gifEmbed {
+  border-left: 0.6rem solid;
+  border-right: 0.6rem solid;
+  border-radius: 0.2rem;
+  height: auto;
+  width: auto;
+
+  div {
+    img {
```
Comment (ketphan02) [inline src/components/App/FeedbackGif/FeedbackGif.module.scss:11]: is this img the same as Image of next/image

File: src/components/Global/LogStatusButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Button } from '../index';
+import { signIn, signOut } from 'next-auth/react';
+import React from 'react';
+
+const LogStatusButton = ({ status }) => {
+  return status === 'authenticated' ? (
+    <Button style={'flat'} content={'Logout'} onClick={() => signOut()} />
+  ) : (
+    <Button style={'flat'} content={'Login'} onClick={() => signIn()} />
```
Comment (ketphan02) [inline src/components/Global/LogStatusButton/index.tsx:9]: I think we should put a TODO: redirect to /auth/login for all functions like this

File: src/pages/auth/login/index.tsx [hunk 3]
```diff
@@ -26,7 +28,8 @@ export default function LoginPage({ providers, csrfToken }) {
       <div className={styles.mainSec}>
         <h2 className={styles.title}>Login</h2>
         {error && <SignInError error={error} />}
-        <form method="post" action="/api/auth/callback/credentials"> {/*TODO: save session into db*/}
+        <form method="post" action={'/api/auth/callback/credentials'}>
```
Comment (ketphan02) [inline src/pages/auth/login/index.tsx:31]: i think we should add a login service here

**Review by ketphan02 — CHANGES_REQUESTED** [review]

File: src/components/App/FeedbackGif/FeedbackGif.module.scss (hunk 1 above)
Comment (Opeyem1a) [reply src/components/App/FeedbackGif/FeedbackGif.module.scss:11]: yep

File: src/pages/auth/login/index.tsx (hunk 3 above)
Comment (Opeyem1a) [reply src/pages/auth/login/index.tsx:31]: What do you mean?

File: src/components/Global/LogStatusButton/index.tsx (hunk 2 above)
Comment (Opeyem1a) [reply src/components/Global/LogStatusButton/index.tsx:9]: Like redirect instead of the `signIn()`/`signOut()`?

**Review by MtheDV — APPROVED** [review]

**Pull request merged**